appinventor/appengine/src/com/google/appinventor/server/LoginServlet.java

```diff
@@ -225,10 +247,80 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
       out.println("<p>" + bundle.getString("requestinstructions") + "</p>\n");
       out.println("<form method=POST action=\"" + req.getRequestURI() + "\">\n");
       out.println(bundle.getString("enteremailaddress") + ":&nbsp;<input type=text name=email value=\"\" size=\"35\"><br />\n");
+      out.println("<input type=hidden name=locale value=\"" + locale + "\">");
```

Comment:
It sets it to the string "null". I'm not going to worry about it now, The whole Send Link functionality doesn't currently work in the public release. Or more to the point, it requires a helper server (because we cannot send mail from Appengine) and that server hasn't been operational for quite a while. The code use in the stand-alone and scaleable code does work because it can send mail itself and doesn't use the helper server.